src/TinyGsmClientA7670.h

```diff
@@ -508,7 +508,7 @@ class TinyGsmA7670 : public TinyGsmModem<TinyGsmA7670>,
       waitResponse();
     }
     sendAT(GF("+CGNSSPWR=1"));  
-    if (waitResponse(10000UL,"+CGNSSPWR: READY!") != 1) { return false; }
+    if (waitResponse(10000UL, GF("+CGNSSPWR: READY!")) != 1) { return false; }
```

Comment:
Compilation failure here: I guess you meant `GF` ? Function called was the one with a String buffer for the returned value.